org.raspberrypi.rpi-imager.yaml

```diff
@@ -0,0 +1,31 @@
+app-id: org.raspberrypi.rpi-imager
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+command: rpi-imager
+rename-desktop-file: rpi-imager.desktop
+rename-icon: rpi-imager
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
```

Comment:
You can swap this to `--socket=fallback-x11` if you don't need the Xwayland socket when running on Wayland.

Reply:
Done, thanks for your suggestion.